src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1713,6 +1695,85 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
     }
 
+    void ControlCore::PersistToPath(const wchar_t* path) const
+    {
+        const wil::unique_handle file{ CreateFileW(path, GENERIC_WRITE, FILE_SHARE_READ | FILE_SHARE_DELETE, nullptr, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, nullptr) };
+        THROW_LAST_ERROR_IF(!file);
+
+        static constexpr size_t writeThreshold = 32 * 1024;

```

Comment:
Since we're writing UTF-16, this results in 64KiB `WriteFile` calls, which I felt like is an ok size to use. Any size is probably fine, as long as we don't call `WriteFile` for each individual `ROW` in the buffer...